### PR TITLE
Enhance LLDP test case for multi-asic platforms

### PR DESCRIPTION
**Pull request by abdosi**
What I did:

This PR is continuation of https://github.com/Azure/sonic-mgmt/pull/2269 where infra changes were done to support multi-asic platforms. In this PR:-

- Fix Issue done in https://github.com/Azure/sonic-mgmt/pull/2269 where `asic_id` is 0.

-  LLDP test is enhanced to run on multi-asic devices.

How I verify:

Verify on both single and multi-asic platforms.

### CODE REVIEW

**Comment by abdosi** [comment]
@arlakshm  can you please help to review this.

File: tests/lldp/test_lldp.py [hunk 1]
```diff
@@ -17,13 +17,13 @@ def lldp_setup(duthosts, rand_one_dut_hostname, patch_lldpctl, unpatch_lldpctl,
...
-def test_lldp(duthosts, rand_one_dut_hostname, localhost, collect_techsupport):
+def test_lldp(duthosts, rand_one_dut_hostname, localhost, collect_techsupport, enum_frontend_asic_index):
     """ verify the LLDP message on DUT """
     duthost = duthosts[rand_one_dut_hostname]
 
-    config_facts  = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
-    lldpctl_facts = duthost.lldpctl_facts(skip_interface_pattern_list=["eth0"])['ansible_facts']
-
+    namespace = duthost.get_namespace_from_asic_id(enum_frontend_asic_index)
+    config_facts = duthost.config_facts(host=duthost.hostname, source="running",namespace=namespace)['ansible_facts']
```
Comment (arlakshm) [inline tests/lldp/test_lldp.py:25]: can we use the sonicasic.config_facts here ?

Comment (abdosi) [reply tests/lldp/test_lldp.py]: @arlakshm Updated.

**Review by arlakshm — APPROVED** [review]

**Pull request merged**